cypress/e2e/talks-interaction.cy.ts

```diff
@@ -0,0 +1,238 @@
+/// <reference types='cypress' />
+
+describe("Talks page correct content", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.get(".navbar").contains("Talks").click();
+    cy.get(".section").contains("Git workflow").as("git-pl");
+    cy.get(".section")
+      .contains(
+        "Steem i Steemit - zdecentralizowane social media na blockchain"
+      )
+      .as("steem-pl");
+    cy.get(".section")
+      .contains("How to use and store your Bitcoins in a secure way")
+      .as("bitcoins-en");
+    cy.get(".section")
+      .contains("Blockchain - czym jest i jak działa")
+      .as("blockchain2-pl");
+    cy.get(".section")
+      .contains("Blockchain - how it works")
+      .as("blockchain1-pl");
+    cy.get(".section").contains("Lightning Network").as("lightning-en");
+    cy.get(".section")
+      .contains("Docker - Easy Containerization")
+      .as("docker-pl");
+    cy.get('#talk-filters > [data-filter="*"]').should("have.class", "active");
+    cy.get('[data-filter="git"]').should("not.have.class", "active");
+    cy.get('[data-filter="docker"]').should("not.have.class", "active");
+    cy.get('[data-filter="blockchain"]').should("not.have.class", "active");
+    cy.get('#language-filters > [data-filter="*"]').should(
+      "have.class",
+      "active"
+    );
+    cy.get('[data-filter="english"]').should("not.have.class", "active");
+    cy.get('[data-filter="polish"]').should("not.have.class", "active");
```

Comment:
because this is in `beforeEach` those assertions are checked multiple times. Do we really need it?

From reading this section, this could be a separate 

```js
it('should be only "All" filter marked as selected after page is loaded')
```